src/start.py

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env python3
+
+import datetime
+import logging
+import os
+import os.path
+import sys
+
+
+class Launcher:
+    def __init__(self):
+        self.logger = ...
+        self.file_handler = ...
+        self.init_logger()
+        self.logger.info("Starting server...")
+
+    def init_logger(self):
+        self.logger = logging.getLogger(__name__)
+        self.logger.setLevel(logging.DEBUG)
+
+        formatter = logging.Formatter("(%(asctime)s.%(msecs)d) [%(levelname)s]: \"%(name)s\": %(message)s",
+                                      "%d-%m-%Y %H:%M:%S")
+        console_handler = logging.StreamHandler()
+        console_handler.setFormatter(formatter)
+        console_handler.setLevel(logging.WARNING)
+        if not os.path.exists("log/"):
+            os.mkdir("log/")
```

Comment:
This is to be discussed, but if we use docker image than it is relatively reasonable to log more or less everything to the console, as when a user just does `docker logs` on a running container and get all the logs from it easily.

Reply:
Another thing is that you can make those (log levels) configurable based on environment variables with your values as defaults and then set it up with the opition `-e` see: https://docs.docker.com/engine/reference/commandline/run/

Reply:
I think I changed that already on the "socket" branch, which is the most up to date... Sorry, I should have told you that.

Making the logs configurable with env vars seems interesting, I'll definitely make it a feature